### DISCUSSION TITLE
Performance...

### DISCUSSION THREAD

**Discussion by lenar** [rekalogika/mapper#101] [General]
Could you give a little opinion about performance of this library? Compared to, for example, jolicode/automapper which touts to be "Very FAST" automapper (page title of https://github.com/jolicode/automapper)? As I understand you preferred clear error messages (good decision most probably) but how much potential performance you lost with that decision? Or not?

**Comment by priyadi** [comment]
I benchmarked all of them a year ago:

![image](https://github.com/user-attachments/assets/5286aaf1-579a-405c-85ee-55fe5a881197)

I consider my mapper on par with the alternatives and plenty fast for my purposes, but YMMV. Jolicode is about 40x faster. If you really need speed, you should consider Jolicode or Micromapper.
